packages/hypernet-core/package.json

```diff
@@ -49,7 +49,7 @@
     "@glazed/did-datastore": "0.2.1",
     "@hypernetlabs/common-repositories": "^0.0.4",
     "@hypernetlabs/gateway-connector": "^0.0.12",
-    "@hypernetlabs/governance-sdk": "^0.0.4",
+    "@hypernetlabs/governance-sdk": "^0.0.6",
```

Comment:
This version doesn't match the 0.0.7 in governance-sdk/package.json, but honestly it should work anyway